src/helpers/observer.js

```diff
@@ -36,7 +52,12 @@ export function observe(draggedEl, dropZones, intervalMs = INTERVAL_MS) {
      */
     function andNow() {
         const currentCenterOfDragged = findCenterOfElement(draggedEl);
-        const scrolled = scrollIfNeeded(currentCenterOfDragged, lastDropZoneFound);
+        const scrollableContainers = findScrollableContainers(lastDropZoneFound);
+        let scrolled = false;
+        for (const container of scrollableContainers) {
+            // scroll only the first container that needs to be scrolled
```

Comment:
if only the first element needs to be scrolled why not `break` after you found one that returned true instead of letting the loop continue to run? I understand your code does the same implicitly but why not make it explicit?
(`if (scrolled) break` or something like that)